semeio/jobs/scripts/gendata_rft.py

```diff
@@ -100,6 +100,19 @@ def _build_parser():
 a specific pressure point is valid
 """,
     )
+    parser.add_argument(
+        "-d",
+        "--directory",
+        required=False,
+        type=str,
+        default=".",
+        help=(
+            "Output directory used for outputting resulting files "
+            "(except for the csvfile output). "
```

Comment:
Is there a reason for not also putting the csv file in the output directory?

Reply:
Not a very good reason, but the proposed feature is there to not clutter up the RUNPATH by tens of files, for which the CSV export is not to blame.

The CSV output file has its own dedicated argument, and there the user can specify an output directory already.

If output directory is set, and CSV output filename is an absolute path, how to proceed would be questionable.

Reply:
I see your point! :) In general I think it would be cleaner if a script dumps all of its output files into the output directory when applicable. I leave the decision to you ;)

Also, maybe `--output-directory` and `-o` is more intuitive command line options?

Reply:
Yes, I had `-o` first, will change back to that. Little bit long argument in ERT config, and  I don't want to shorten  to  OUTPUT  (to mitigate confusion with the CSV output)